cupy/cudnn.pyx

```diff
@@ -1052,11 +1052,11 @@ def rnn_backward_data(
         cx = core._internal_ascontiguousarray(cx)
     w = core._internal_ascontiguousarray(w)
     xs = core._internal_ascontiguousarray(xs)
-    ys = core._internal_ascontiguousarray(ys)
+    ys = core._internal_ascontiguousarray(ys.copy())
```

Comment:
Currently, I create views of `ys` and `dys`.
More suitable solution seems to be creating views of `ys` and `dys`.